core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -24,15 +24,120 @@ coreos:
         [Install]
         RequiredBy=format-etcd2-volume.service
 
-    - name: etcd2.service
+    - name: etcdadm-reconfigure.service
+      enable: true
+      content: |
+        [Unit]
+        Description=etcdadm reconfigure runner
+        BindsTo=etcd-member.service
+        Before=etcd-member.service
+        Wants=cfn-etcd-environment.service
+        After=cfn-etcd-environment.service
+        Wants=network.target
+
+        [Service]
+        Type=oneshot
+        RemainAfterExit=yes
+        RestartSec=5
```

Comment:
No! Good catch.
Btw, I wanted the reconfiguration to retry until it eventually succeeds. However, setting it to  `Type=simple` breaks `etcd-member.service`. I hope there's any way to easily make it restart-able.